src/checkovRunner.ts

```diff
@@ -51,8 +51,8 @@ const getDockerRunParams = (workspaceRoot: string | undefined, filePath: string,
     
     return configFilePath ?
         [...params, '-v', `"${path.dirname(configFilePath)}:${configMountDir}"`, image, 
-            '--config-file', `${configMountDir}/${path.basename(configFilePath)}`, '-f', filePathToScan] :
-        [...params, image, '-f', filePathToScan];
+            '--config-file', `${configMountDir}/${path.basename(configFilePath)}`, '-f', `"${filePathToScan}"`] :
+        [...params, image, '-f', `"filePathToScan"`];
```

Comment:
Besides doing what I'm suggesting below, you could also change directly the `convertToUnixPath` function return
```suggestion
        [...params, image, '-f', `"${filePathToScan}"`];
```

Reply:
I've made the change to convertToUnixPath function to wrap in there. if we do it there we don't need to do it within the getDockerRunParams function directly, otherwise they would be double quoted.